docs/webrtc/nanostream_webrtc_screen_sharing.md

```diff
@@ -4,15 +4,17 @@ title: Screen Sharing
 sidebar_label: Screen Sharing
 ---
 
-## Screen Sharing 
+The `nanoStream Webcaster` supports screen sharing.
```

Comment:
Hm... do we highlight `nanoStream Webcaster` in the rest of the text?

Reply:
no, changed